src/fragments/lib-v1/project-setup/android/create-application/10_createProject.mdx

```diff
@@ -1,10 +1,10 @@
 **Open Android Studio.**  Select **+ Create New Project.**
 
-![](/images/lib/getting-started/android/set-up-android-studio-welcome.png)
+![Android studio welcome page](/images/lib/getting-started/android/set-up-android-studio-welcome.png)
```

Comment:
Let's be consistent and add "shows" as well? Also are other accessibility texts using the same sentence structure?

Reply:
Agreed.

Reply:
Done